bugsnag-android-core/src/main/java/com/bugsnag/android/internal/journal/Journal.kt

```diff
@@ -97,7 +97,10 @@ class Journal(val type: String, val version: Int) {
          */
         @Throws(IOException::class)
         fun serialize(out: OutputStream) {
-            JsonHelper.serialize(mutableMapOf(path to value), out)
+            // optimization - avoid unnecessary instantiation of Pair() on a hot path
```

Comment:
I was actually touching the same line for a different PR. Can you try comparing it with just changing to `JsonHelper.serialize(mapOf(path to value), out)`? Since it's just being serialized, it doesn't need to be mutable. When I did this, it pretty much disappeared from the flame graph in my tests.

Reply:
`mapOf(path to value)` is marginally faster - I'll update the PR to use that.

<img width="728" alt="Screenshot 2021-11-09 at 15 13 57" src="https://user-images.githubusercontent.com/11800640/140950907-cb02454a-e3e8-4992-bfdb-34a240ac6fcb.png">